src/main/java/core/basesyntax/model/Check.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.model;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+public class Check {
+    LocalDateTime operationTime;
+    String bank;
+    String purchaseList;
+    BigDecimal discount;
+    BigDecimal totalAmount;
```

Comment:
let's create an easier model for now
for example, User with the fields `name` and `age`

the example of user input in ConsoleHandler will be `200 0.6 Masha 20`